iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -581,6 +581,14 @@ + (id)initWithLaunchOptions:(NSDictionary*)launchOptions
     return self;
 }
 
++ (void)setAppId:(NSString*)appId {
+    
```

Comment:
is ok to have this methods empty?

Reply:
In another PR, just added this empty method so I could create a PR with this branch